config/bootstrap.php

```diff
@@ -33,11 +33,21 @@
  */
 require CORE_PATH . 'config' . DS . 'bootstrap.php';
 
+// You can remove this if you are confident that your PHP version is sufficient.
+if (version_compare(PHP_VERSION, '5.5.9') < 0) {
```

Comment:
Stealing my thoughts (in terms of multiple places).
- composer.json
- boostrap
- docs

I completely agree.
However things should read this from boostrap.php somehow. Why? Here it should be as fast as possible.
Reading from the composer.json in boostrap sounds... like slow. :(.
Another option would be to somehow have some pseudo-constant on github ... if that's possible?

It's the same for non default extensions.